docs/userguide/networking/work-with-networks.md

```diff
@@ -499,6 +499,130 @@ environment variables into a running container without significant effort and he
 it is not compatible with `docker network` which provides a dynamic way to connect/
 disconnect containers to/from a network.
 
+### Network-scoped alias
+
+While `links` provide private name resolution that is localized within a container,
+the network-scoped alias provides a way for a container to be discovered by an
+alternate name by any other container within the scope of a particular network.
+Unlike the `link` alias, which is defined by the consumer of a service, the
```

Comment:
Not really. `--link` always called it an alias : `--link=""  : Add link to another container (<name or id>:alias or <name or id>)` as in `https://github.com/docker/docker/blob/master/docs/reference/run.md` and many other places.